src/learner.cc

```diff
@@ -306,18 +328,26 @@ struct LearnerTrainParam : public XGBoostParameter<LearnerTrainParam> {
   // specified by users.  Move them to model parameter once we can get rid of binary IO.
   std::string booster;
   std::string objective;
+  // This is a training parameter and is not saved (nor loaded) in the model.
+  Strategy multi_strategy{Strategy::kComposite};
 
   // declare parameters
   DMLC_DECLARE_PARAMETER(LearnerTrainParam) {
     DMLC_DECLARE_FIELD(disable_default_eval_metric)
         .set_default(false)
         .describe("Flag to disable default metric. Set to >0 to disable");
-    DMLC_DECLARE_FIELD(booster)
-        .set_default("gbtree")
-        .describe("Gradient booster used for training.");
+    DMLC_DECLARE_FIELD(booster).set_default("gbtree").describe(
+        "Gradient booster used for training.");
     DMLC_DECLARE_FIELD(objective)
         .set_default("reg:squarederror")
         .describe("Objective function used for obtaining gradient.");
+    DMLC_DECLARE_FIELD(multi_strategy)
+        .add_enum("compo", Strategy::kComposite)
+        .add_enum("mono", Strategy::kMono)
```

Comment:
```suggestion
        .add_enum("composite", Strategy::kComposite)
        .add_enum("monolithic", Strategy::kMonolithic)
```
Why not spell them out fully? Given the wide use of IDEs, we can afford to be a bit verbose when it comes to naming.

Reply:
Done.